src/gui_common/ScreenFade.cs

```diff
@@ -56,18 +80,16 @@ public void FadeToWhite()
 
     public void FadeTo(Color initial, Color final)
     {
-        Rect.Color = initial;
+        rect.Color = initial;
 
-        Fader.InterpolateProperty(Rect, "color", initial, final, FadeDuration);
+        fader.InterpolateProperty(rect, "color", initial, final, FadeDuration);
 
-        Fader.Start();
+        fader.Start();
     }
 
     public void OnStarted()
     {
-        ControlNode.Show();
-
-        switch (FadeTransition)
+        switch (CurrentFadeType)
         {
             case FadeType.FadeIn:
```

Comment:
I suppose maybe the entire system is backwards ie. to me fade *in* is when the gameplay becomes visible, and fade out is when the gameplay fades out to black...

Reply:
Ah yeah I've just looked this up now and it seems you're right. Somehow I confused this when implementing the screen fades a while, whoops..